iteration.qmd

```diff
@@ -421,18 +441,19 @@ data <- bind_rows(data2019, data2020, data2021, data2022)
 
 You can imagine that this would get tedious quickly, especially if you had hundreds of files, not just four.
 The following sections show you how to automate this sort of task.
-There are three basic steps: use `dir()` list all the files in a directory, then use `purrr::map()` to read each of them into a list, then use `purrr::list_rbind()` to combine them into a single data frame.
+There are three basic steps: use `dir()` to list all the files in a directory, then use `purrr::map()` to read each of them into a list, then use `purrr::list_rbind()` to combine them into a single data frame.
 We'll then discuss how you can handle situations of increasing heterogeneity, where you can't do exactly the same thing to every file.
 
 ### Listing files in a directory
 
 `dir()` lists the files in a directory.
+TO CONSIDER: why not use it via the more obvious name `list.files()`?
```

Comment:
If we're not going to plug fs, then I think I'd use `list.files()` over `dir()`.